app/config/index.js

```diff
@@ -1,28 +1,19 @@
 const Joi = require('joi')
 const authConfig = require('./auth')
+const cacheConfig = require('./cache')
+const { DEVELOPMENT, TEST, PRODUCTION } = require('../constants/environments')
 
 // Define config schema
 const schema = Joi.object({
   serviceName: Joi.string().default('Dangerous Dogs Index'),
   port: Joi.number().default(3001),
   env: Joi.string().valid('development', 'test', 'production').default('development'),
-  useRedis: Joi.boolean().default(false),
   ddiIndexApi: {
     baseUrl: Joi.string().required()
   },
   ddiEventsApi: {
     baseUrl: Joi.string().required()
   },
-  cache: {
-    expiresIn: Joi.number().default(1000 * 3600 * 24 * 3), // 3 days
-    options: {
-      host: Joi.string().default('redis-hostname.default'),
-      partition: Joi.string().default('dangerous-dog-act-portal'),
-      password: Joi.string().allow(''),
-      port: Joi.number().default(6379),
-      tls: Joi.object()
-    }
-  },
   cookie: {
     cookieNameCookiePolicy: Joi.string().default('dangerous_dog_act_portal_cookie_policy'),
```

Comment:
I know this isn't really part of setting up Redis but I think we should change the cookie names so as not to conflict with portal sessions. These two names will need replacing across the application to something specific to enforcement

Reply:
another good catch